src/sagemaker/workflow/functions.py

```diff
@@ -54,3 +55,36 @@ def expr(self):
                 ],
             },
         }
+
+
+@attr.s
```

Comment:
For this class, we are introducing the same duplicated block of code here as in condition_step.py, while marking the block there as deprecated. The class itself is the same. If we still plan to reuse the same code in condition_step as well as here, why can't we pull this out somewhere common, like utils, where both can import it